src/logservice/logminer/ob_log_miner_logger.cpp

```diff
@@ -50,25 +52,40 @@ int LogMinerLogger::log_progress(int64_t record_num, int64_t current_ts, int64_t
   int ret = OB_SUCCESS;
   double percentage = double(current_ts - begin_ts) / double(end_ts - begin_ts);
   double progress = 0;
+  struct winsize w;
```

Comment:
It is better to initialize it, otherwise strange random results may occur due to possible ioctl failures.